package.json

```diff
@@ -32,8 +32,9 @@
     "gpsoauthnode": "0.0.5",
     "long": "^3.2.0",
     "node-pogo-protos": "^1.2.0",
-    "request": "^2.74.0",
-    "s2-geometry": "^1.2.7"
+    "request": "^2.73.0",
```

Comment:
Looks like I'll manually merge this PR and my WIP implementation into one anyway (while giving @SpencerSharkey credit, of course) so don't worry about it.